pkg/odo/cli/component/create.go

```diff
@@ -220,8 +222,8 @@ A full list of component types that can be deployed is available using: 'odo cat
 			odoutil.LogErrorAndExit(err, "")
 
 			// Git is the only one using BuildConfig since we need to retrieve the git
-			err = component.Build(client, componentName, applicationName, true, stdout)
-			odoutil.LogErrorAndExit(err, "")
+			err = component.Build(client, componentName, applicationName, wait, stdout)
+			odoutil.CheckError(err, "")
```

Comment:
@syamgk This line causing the CI failure. Please have a look

Reply:
yep thanks, fixed